docs/content/ingestion/schema-changes.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: doc_page
+---
+# Schema Changes
+
+Schemas for datasources can change at any time and Druid supports different schemas among segments.
+
+## Replacing Segments
+
+Druid uniquely 
+identifies segments using the datasource, interval, version, and partition number. The partition number is only present if 
+there are multiple segments created for some granularity of time. For example, if you have hourly segments, but you 
+have more data in an hour than a single segment can hold, you can create multiple segments for the same hour. These segments will share 
+the same datasource, interval, and version, but have linearly increasing partition numbers.
+
+```
+foo_2015-01-01/2015-01-02_v1_0
```

Comment:
maybe worth making it explicit that dataSource = foo, interval = 2015-01-01/2015-01-02, version = v1, partitionNum = 0